include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1714,6 +1714,33 @@ int bt_le_ext_adv_delete(struct bt_le_ext_adv *adv);
  */
 uint8_t bt_le_ext_adv_get_index(struct bt_le_ext_adv *adv);
 
+/** Advertising states */
+enum bt_le_ext_adv_state {
+	/** No state */
+	BT_LE_EXT_ADV_STATE_NONE,
```

Comment:
Is this "no state" as in an invalid value?

Reply:
It's actually unused, so perhaps the starting value of the enum should be `BT_LE_EXT_ADV_STATE_CREATED`

Reply:
Continuing my question from https://github.com/zephyrproject-rtos/zephyr/pull/90185#discussion_r2126162620

Now there is only two values, 0 or 1, please use a boolean instead.

Reply:
The enum is far more extendable if we ever want to add back a state like `PAUSED` if that becomes a necessity.